setup.py

```diff
@@ -0,0 +1,9 @@
+"""setup file"""
+from distutils.core import setup
+
+setup(
+    name='sem_size_analysis',
+    version='0.1',
+    packages=['src', ],
+    long_description=open('README.md').read()
```

Comment:
You need to configure your setup.py to use your requirements.txt, as it won't do this by default. One of the examples I sent you illustrates how to do this.